pkg/k8s/k8s.go

```diff
@@ -15,20 +15,28 @@ const (
 	ReplicationControllers = "replicationcontrollers"
 	Services               = "services"
 	All                    = "all"
+	Authority              = "authority"
```

Comment:
Ah, also for consistency with the other resources, I'd prefer to call this const `Authorities` instead of `Authority`.